default-templates/new-account/index.html

```diff
@@ -54,26 +53,44 @@ <h1>Apps</h1>
       </a>
     </div>
   </section>
+
+  <section class="hidden" id="account-settings">
+    <h1>Account settings</h1>
+    <div class="list-group">
+      <a href="/account/delete/" class="list-group-item">
+        <span class="lead">Delete account</span>
+      </a>
+    </div>
+  </section>
 </div>
 
 <script src="/common/js/solid-auth-client.bundle.js"></script>
 <script type="text/javascript">
   (function () {
-    'use strict'
-    var button = document.getElementById('session-action')
-    var loggedIn = false
+    'use strict';
+    var button = document.getElementById('session-action');
+    var loggedIn = false;
+    var accountSettings = document.getElementById('account-settings');
 
-    solid.auth.trackSession(session => {
-      loggedIn = !!session
-      button.innerText = loggedIn ? 'Log out' : 'Log in'
+    solid.auth.trackSession(function(session) {
+      loggedIn = !!session;
```

Comment:
Why all the semicolons here? :) Our style is to do without.. (Apologies if this is side effects from another PR)